### ISSUE TITLE
Support Websocket API requests

### ISSUE THREAD

**Issue by suceava** [dbartholomae/middy-middleware-jwt-auth#160]
**Is your feature request related to a problem? Please describe.**
I am setting up a Websocket API in API Gateway and I'm trying to use `middy` with this package to get the JWT token.  Currently this code will throw a 401 'No valid bearer token was set in the authorization header' because when it checks for an authorized event, the event doesn't have all the properties expected for a REST or Http API - it's missing both the `event.httpMethod` property as well as the `event.requestContext.http` property.
Instead, the event has a few other properties that can be checked against: e.g. `event.requestContext.connectionId` and `event.requestContext.eventType`

** Would you be willing to help with a PR? **
- [x] Yes, absolutely
- [ ] Yes, with some guidance
- [ ] Unfortunately no time :'-(

I have a PR ready and will submit it shortly.

## Describe the solution you'd like
I would like the package to validate the event coming from the API Gateway Websocket API and then proceed to validate the JWT token

## Additional context
Here is a sample event for a Websocket API:
```
{
  headers: {
    Authorization: 'Bearer <token>',
    Host: 'xxx.execute-api.us-west-1.amazonaws.com',
    'Sec-WebSocket-Extensions': 'permessage-deflate; client_max_window_bits',
    'Sec-WebSocket-Key': 'oNeTquURyp1MdCr+f8iI/A==',
    'Sec-WebSocket-Version': '13',
    'X-Amzn-Trace-Id': 'Root=1-624c5156-5421759b1bf059522709fec2',
    'X-Forwarded-For': '127.0.0.1',
    'X-Forwarded-Port': '443',
    'X-Forwarded-Proto': 'https'
  },
  multiValueHeaders: {
    Authorization: [
      'Bearer <token>'
    ],
    Host: [ 'xxx.execute-api.us-west-1.amazonaws.com' ],
    'Sec-WebSocket-Extensions': [ 'permessage-deflate; client_max_window_bits' ],
    'Sec-WebSocket-Key': [ 'oNeTquURyp1MdCr+f8iI/A==' ],
    'Sec-WebSocket-Version': [ '13' ],
    'X-Amzn-Trace-Id': [ 'Root=1-624c5156-5421759b1bf059522709fec2' ],
    'X-Forwarded-For': [ '127.0.0.1' ],
    'X-Forwarded-Port': [ '443' ],
    'X-Forwarded-Proto': [ 'https' ]
  },
  requestContext: {
    apiId: 'xxx',
    connectedAt: 1649168726649,
    connectionId: 'QHGlkfOHyK4CJQA=',
    domainName: 'xxx.execute-api.us-west-1.amazonaws.com',
    eventType: 'CONNECT',
    extendedRequestId: 'QHGlkHM3yK4FgJQ=',
    identity: { sourceIp: '127.0.0.1' },
    messageDirection: 'IN',
    requestId: 'QHGlkHM3yK4FgJQ=',
    requestTime: '05/Apr/2022:14:25:26 +0000',
    requestTimeEpoch: 1649168726650,
    routeKey: '$connect',
    stage: 'local'
  },
  isBase64Encoded: false
}
```

**Comment by dbartholomae** [comment]
Thanks, makes sense! :)

**Comment by dbartholomae** [comment]
Published in 5.1.0
